src/app/components/card/card.component.ts

```diff
@@ -0,0 +1,11 @@
+import { Component, Input } from '@angular/core';
+import Pokemon from '../../models/Pokemon';
+
+@Component({
+  selector: 'app-card',
+  templateUrl: './card.component.html',
+  styleUrls: ['./card.component.css']
```

Comment:
in tutti i componenti abituati sempre a mettere changeDetectionStrategy: OnPush, sempre
di default non lo mettono ma in realtà in termini di performance bisognerebbe farlo

vedi cos'è perchè molto molto importante